backend/src/nodes/impl/rembg/pymatting/cf_laplacian.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+import scipy.sparse
+from numba import njit
```

Comment:
This looks like another dependency.

Reply:
Ah crap, that's what I get for forgetting to test with `npm start`.